lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/migration/MigratableTableDAOImpl.java

```diff
@@ -225,6 +229,25 @@ private void registerObject(MigratableDatabaseObject dbo, boolean isRoot) {
 		registeredMigrationTypes.add(dbo.getMigratableTableType());
 
 	}
+
+	private void validateEtagColumn(String tableName, String columnName) {
+		try (
```

Comment:
Please implement using jdbcTemplate and a query against INFORMATION_SCHEMA, as you showed me in mySQLWorkbench.